overlord/overlord_test.go

```diff
@@ -80,6 +81,11 @@ func fakePruneTicker() (w *ticker, restore func()) {
 }
 
 func (ovs *overlordSuite) SetUpTest(c *C) {
+	// temporary: skip due to timeouts on riscv64
+	if runtime.GOARCH == "riscv64" {
```

Comment:
hm maybe we could set an env flag to skip those tests during the build? I sometimes saw those tests fail to OBS too (probably running in some low-resource VMs) and would happily reuse the flag if possible.

Reply:
How about `if runtime.GOARCH == "riscv64" || os.Getenv("SKIP_SLOW_TESTS") != "" {` ?

Reply:
SNAPD_SKIP_SLOW_TESTS maybe?

Reply:
Done